### DISCUSSION TITLE
Filter on multi element

### DISCUSSION THREAD

**Discussion by MuhammadRahman-awin** [orchestral/parser#101] [Q&A]
First, THANKS A LOT for creating this. However, when I am trying to implement a filter on 
` 'users' => ['uses' => 'collection.user[id,name]']` I am really unsure how to do that.
Basically I want to make sure my data is **trimmed** using a `trimFilter` 

Will you be able to update.

Thanks.

**Comment by whoishardik** [comment]
Anyone got solution? I have same problem.
